modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -171,9 +199,46 @@ Events:     <11>
 <3> The labels applied to the node.
 <4> The annotations applied to the node.
 <5> The taints applied to the node.
-<6> Node conditions.
+<6> The node conditions and status. The `conditions` stanza lists the `Ready`, 'PIDPressure`, 'PIDPressure`, 'MemoryPressure`, 'DiskPressure` and `OutOfDisk` status. These condition are described later in this section.
```

Comment:
Some of the ` here are ' instead, so output is rendering normal text. Also should have serial comma after DiskPressure.